js/app.js

```diff
@@ -183,6 +183,18 @@ function verifyArguments( args ) {
  */
 
 /**
+ * @callback YoastSEO.App~updatedContentResults
+ *
+ * @param {object} results The updated page analysis results.
```

Comment:
The overall score also needs to be documented as a param.

As for the order of the arguments, I would put the overal score first. Less specific first.